docs/pages/installation/reverse-proxy-configuration.rst

```diff
@@ -0,0 +1,169 @@
+
+.. code-block:: xml
+
+    <VirtualHost *:443>
+    ServerName proxy-server
+      # activate HTTPS server certificate on the reverse proxy
+        SSLEngine on
+        SSLCertificateFile "<Apache_home>/conf/proxy-server.crt"
+        SSLCertificateKeyFile "<Apache_home>/conf/proxy-server.key"
+        SSLCertificateChainFile "<Apache_home>/conf/proxy-server-ca.crt"
+      # activate the client certificate authentication
+        SSLCACertificateFile "/etc/nuts/data/truststore.pem"
+        SSLVerifyClient require
+        SSLVerifyDepth 1
+        SSLProxyEngine On
+      # initialize the special headers to a blank value to avoid http header forgeries
+        RequestHeader set SSL_CLIENT_CERT ""
+        <Location /api/fhir>
+            # forward certificate for acceptance list checking
+            RequestHeader set SSL_CLIENT_CERT "%{SSL_CLIENT_CERT}s"
+            ProxyPass http://localhost:8080
+            ProxyPassReverse http://localhost:8080
+        </Location>
+    </VirtualHost>
+
+HAProxy
+=======
+
+.. note::
+
+    help requested on a valid HAProxy example
+
+Nginx
+=====
+
+Used nginx version: 1.17.0
+
+.. code-block:: text
+
+    http {
+      # only TLS > 1.2 is acceptable
+      ssl_protocols TLSv1.2 TLSv1.3;
+      ssl_prefer_server_ciphers on;
+
+      access_log /var/log/nginx/access.log;
+      error_log /var/log/nginx/error.log;
+
+      # server on port 80 for HTTP -> HTTPS redirect
+      server {
+        listen 80;
+        server_name example.com;
+        return 301 https://example.com$request_uri;
+      }
+
+      # The HTTPS server, which proxies our requests
+      server {
+        listen 443 ssl;
+        server_name example.com;
+
+        ssl_protocols TLSv1.2 TLSv1.3;
+        # server certificate
+        ssl_certificate /etc/nginx/ssl/example.com/fullchain.pem;
+        ssl_certificate_key /etc/nginx/ssl/example.com/privkey.pem;
+
+        # client certificate,
+        # here we use the exported truststore.pem in the case Nuts is
+        # running on the same machine
+        ssl_trusted_certificate /etc/nuts/data/truststore.pem;
+        # the client certificate is optional, so for example status endpoints can be accessed
+        ssl_verify_client optional;
+
+        access_log /var/log/nginx/example.com;
+
+        location /api/fhir {
+          # make the certificate non-optional for this path
+          # if the client-side certificate failed to authenticate, show a 403
+          # message to the client
+          if ($ssl_client_verify != SUCCESS) {
+            return 403;
+          }
+
+          proxy_set_header        Host $host;
+          proxy_set_header        X-Real-IP $remote_addr;
+          proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
+          proxy_set_header        X-Forwarded-Proto $scheme;
+          # forward certificate for acceptance list checking
+          proxy_set_header        X-Ssl-Client-Cert $ssl_client_cert;
```

Comment:
These headers differ from Apache example.

Reply:
Intentionally since it appears apache and nginx 'prefer' different styles so I kept the examples in style with docs from respective proxies.